### DISCUSSION TITLE
new icon?!

### DISCUSSION THREAD

**Discussion by ibyteyou** [atom/atom#22816] [General]
<img width="121" alt="Снимок экрана 2021-08-05 в 22 32 35" src="https://user-images.githubusercontent.com/16253399/128409973-02ebcc79-6dc8-43c2-abe2-64e34a1ac604.png">
whaaaaat?

**Comment by icecream17** [comment]
Maybe see this: https://github.com/atom/atom/pull/22315?
it was also mentioned is the release notes https://github.com/atom/atom/releases/tag/v1.58.0
